canonical/jaas-dashboard
# WD-2736 - Scroll to config errors

## Done

- Scroll to config errors when they're off screen.
- Migrate SlidePanel to TypeScript.
- Close the config panel on success.

## QA

- Pull code
- Run `dotrun clean && dotrun serve`
- Open http://0.0.0.0:8036/
- Go to the config panel for an app with lots of options.
- Scroll down and enter an invalid value (e.g. you could inspect a number field and change the field type to text and then enter a non number character).
- Save the form and you should get scrolled up to the error.
- Enter all valid values and submit again.
- This time the panel should close once the request is successful.

## Details

https://warthogs.atlassian.net/browse/WD-2736
Fixes: #1418.
